source/_integrations/shelly.markdown

```diff
@@ -125,6 +125,7 @@ For generation 2 hardware it's possible to select if a device's input is connect
 ## Event entities (generation 1)
 
 If the **BUTTON TYPE** of the switch connected to the device is set to `momentary` or `detached switch`, the integration creates an event entity for this switch. You can use this entity in your automations.
+(Important to note: for relay devices, this is not the same as the binary sensor - it doesn't track the state of a relay, it only fires events when a detached switch is turned from `off` to `on`, and doesn't fire when a detached switch is turned from `on` to `off`)
```

Comment:
IMO this statement is not true. Both the event entities and `shelly.click` events inform about actions performed on the button (momentary button, not switch) and are not related to the state of the relay. If you use a bistable switch in detached mode, you should track its status using the input binary sensor.

Reply:
Okay, please can you suggest an alternative - you've asked me to update the docs, and I've done so based on my experience with Shelly relays (1/1pm and 2.5) - the statement I've made here is _exactly_ the behaviour I've observed with these relays when the switch is set to detached mode.

Reply:
First of all, I didn't ask you to update the documentation.

Currently, there is an information in the documentation about the binary input sensors you should use in your setup.

![obraz](https://github.com/home-assistant/home-assistant.io/assets/478555/ef7903cd-b1a6-4370-a875-b0b8d91f1de6)

Adding information for events/event entities that they do not correspond to changes in relay states is, in my opinion, unnecessary because they are, by design/definition, not related to the relay states, those are "input events". I know I may not be very representative when it comes to understanding the documentation and how Shelly devices work, but the change you proposed would be confusing for users in my opinion.
Perhaps it would be better to add a note that if you are using a toggle (bistable) switch you should use the input binary sensor rather than events/event elements in the automation.

Reply:
Wow, how do people work with you??? lol .. whatever man, I'll close this then

Reply:
Tell me, how is that _*several*_ people made the same mistake I did? we all didn't read the docs, right??
I'm sick of maintainers like you that treat people that want to contribute, like dirt. I will not be attempting to contribute again, if this is the response.

Reply:
I'm abandoning this now, I was making changes that made sense to me, to hopefully help others that had the same issue. I don't have the appetite or endurance to respond here, closing the PR, maybe someone else will hit up against this same issue and submit a more sensible doc update, but that's clearly not me 👍